### ISSUE TITLE
flake8 output

### ISSUE THREAD

**Issue by dericdesta** [IPMI-ICNS-UKE/respiratory-motion-prediction#4]
Here is some helpful advice:
```
./setup.py:9:2: W292 no newline at end of file
) ^
./rmp/eval_model.py:1:21: E999 SyntaxError: invalid syntax
import logging
              
^
./rmp/hyperoptimizer.py:33:19: E999 SyntaxError: invalid syntax
def partial(func, /, *args, **keywords):
                  ^
./rmp/global_config.py:12:32: E261 at least two spaces before inline comment
RESULT_DIR.mkdir(exist_ok=True)# change
                               ^
./rmp/global_config.py:13:63: E261 at least two spaces before inline comment
DATALAKE = Path("/datalake/open_access_rpm_signals_master.db") # change, path to databse
                                                              ^
./rmp/models.py:1:9: E999 SyntaxError: invalid syntax
import math
        ^
./rmp/gym.py:1:15: E999 SyntaxError: invalid syntax
from __future__ import annotations
              ^
./rmp/dataloader.py:1:12: E999 SyntaxError: invalid syntax
"""
Loads respiratory signal and performs preprocessing (smoothing, de-noising, scaling etc.)

"""
   
       ^
./rmp/plotting.py:1:15: E999 SyntaxError: invalid syntax
"""
Simple functions for plotting signals during training and final results.
Also, visualization of attention heads and corresponding tikz code.
"""
   
          ^
./rmp/metrics.py:1:14: E999 SyntaxError: invalid syntax
import logging
             ^
./rmp/Tests/transformer_forward_vs_inference.py:1:18: E999 SyntaxError: invalid syntax
import logging
              
^
./rmp/my_utils/input_paras_check.py:1:22: E999 SyntaxError: invalid syntax
from __future__ import annotations
                     ^
./rmp/my_utils/decorators.py:69:13: VNE001 single letter variable names like 'd' are not allowed
        for d in (args_dict, kwargs_dict):
            ^
./rmp/my_utils/logger.py:46:15: VNE001 single letter variable names like 's' are not allowed
        self, s: str, text_color: Tuple[int, int, int], bold: bool = False
              ^
./rmp/my_utils/logger.py:96:29: VNE001 single letter variable names like 's' are not allowed
    def _shorten_left(self, s: str, max_length: int) -> str:
                            ^
./rmp/my_utils/logger.py:98:13: VNE001 single letter variable names like 's' are not allowed
            s = "..." + s[-(max_length - 3) :]
            ^
./rmp/my_utils/logger.py:193:61: B026 Star-arg unpacking after a keyword argument is strongly discouraged, because it only works when the keyword parameter is declared after all parameters supplied by the unpacked sequence, and this change of ordering can surprise and mislead readers.
        orig_tqdm.tqdm, logger=logger, log_level=log_level, *args, **kwargs
                                                            ^
./rmp/my_utils/logger.py:201:63: B026 Star-arg unpacking after a keyword argument is strongly discouraged, because it only works when the keyword parameter is declared after all parameters supplied by the unpacked sequence, and this change of ordering can surprise and mislead readers.
        orig_tqdm.trange, logger=logger, log_level=log_level, *args, **kwargs
                                                              ^
./scripts/run_eval_model.py:19:40: E261 at least two spaces before inline comment
    evaluater = Eval.init_dlinear_480() # select model and pred horizon here
                                       ^
./scripts/run_eval_model.py:22:1: W391 blank line at end of file

^
./scripts/run_hyperopt.py:42:89: E501 line too long (125 > 88 characters)
    """Creating and checking configuration input before staring hyperparameter search.

    :param model_arch: chosen model architecture
    :param train_batch_size: batch size for training
    :param eval_batch_size: batch size for evaluation, i.e. validation and testing
    :param future_steps : determines the prediction horizon, samples per second = 25 HZ; -> 12: 480ms ; 17: 680ms ; 23: 920ms
    :param train_signal_length_s: determines length of random singal subset (allows stacking/batching signal subsets). If None, entire signal is used.
    :param train_min_length_s: all signals shorter than train_min_length_s will be excluded from train set
    :param test_signal_length_s: all signals shorter than test_signal_length_s will be excluded from val/test set # ToDo
    :return:
    """
                                                                                      

^
./scripts/run_hyperopt.py:43:89: E501 line too long (150 > 88 characters)
    """Creating and checking configuration input before staring hyperparameter search.

    :param model_arch: chosen model architecture
    :param train_batch_size: batch size for training
    :param eval_batch_size: batch size for evaluation, i.e. validation and testing
    :param future_steps : determines the prediction horizon, samples per second = 25 HZ; -> 12: 480ms ; 17: 680ms ; 23: 920ms
    :param train_signal_length_s: determines length of random singal subset (allows stacking/batching signal subsets). If None, entire signal is used.
    :param train_min_length_s: all signals shorter than train_min_length_s will be excluded from train set
    :param test_signal_length_s: all signals shorter than test_signal_length_s will be excluded from val/test set # ToDo
    :return:
    """
                                                                                      

^
./scripts/run_hyperopt.py:44:89: E501 line too long (106 > 88 characters)
    """Creating and checking configuration input before staring hyperparameter search.

    :param model_arch: chosen model architecture
    :param train_batch_size: batch size for training
    :param eval_batch_size: batch size for evaluation, i.e. validation and testing
    :param future_steps : determines the prediction horizon, samples per second = 25 HZ; -> 12: 480ms ; 17: 680ms ; 23: 920ms
    :param train_signal_length_s: determines length of random singal subset (allows stacking/batching signal subsets). If None, entire signal is used.
    :param train_min_length_s: all signals shorter than train_min_length_s will be excluded from train set
    :param test_signal_length_s: all signals shorter than test_signal_length_s will be excluded from val/test set # ToDo
    :return:
    """
                                                                                      

^
./scripts/run_hyperopt.py:45:89: E501 line too long (120 > 88 characters)
    """Creating and checking configuration input before staring hyperparameter search.

    :param model_arch: chosen model architecture
    :param train_batch_size: batch size for training
    :param eval_batch_size: batch size for evaluation, i.e. validation and testing
    :param future_steps : determines the prediction horizon, samples per second = 25 HZ; -> 12: 480ms ; 17: 680ms ; 23: 920ms
    :param train_signal_length_s: determines length of random singal subset (allows stacking/batching signal subsets). If None, entire signal is used.
    :param train_min_length_s: all signals shorter than train_min_length_s will be excluded from train set
    :param test_signal_length_s: all signals shorter than test_signal_length_s will be excluded from val/test set # ToDo
    :return:
    """
                                                                                      

^
./scripts/run_hyperopt.py:83:89: E501 line too long (111 > 88 characters)
        f"Predict the value which is {final_config['future_steps'] * 40}ms ahead (single point prediction). \n"
                                                                                        ^
./scripts/run_hyperopt.py:84:89: E501 line too long (99 > 88 characters)
        f"Input signal are noisy, signal to noise ratio is {final_config['white_noise_db']} dB. \n"
                                                                                        ^
./scripts/run_hyperopt.py:85:89: E501 line too long (104 > 88 characters)
        f"Targets are sampled from corresponding smooth signal: {final_config['use_smooth_target']}. \n"
                                                                                        ^
./scripts/run_hyperopt.py:86:89: E501 line too long (106 > 88 characters)
        f"MinMax scaling is performed on the first {final_config['scaling_period_s']}s of each signal. \n"
                                                                                        ^
2     B026 Star-arg unpacking after a keyword argument is strongly discouraged, because it only works when the keyword parameter is declared after all parameters supplied by the unpacked sequence, and this change of ordering can surprise and mislead readers.
3     E261 at least two spaces before inline comment
8     E501 line too long (125 > 88 characters)
9     E999 SyntaxError: invalid syntax
4     VNE001 single letter variable names like 'd' are not allowed
1     W292 no newline at end of file
1     W391 blank line at end of file
```

**Comment by lukas-wi** [comment]
fixed at  [f459538](https://github.com/IPMI-ICNS-UKE/respiratory-motion-prediction/commit/f4595386317e1e6ae3d7e9129557d6aa88e7d17e)
LG?

**Comment by dericdesta** [comment]
LGTM!
